FogCreek/Glitch-Community
# Lowercase authtype for analytics sign in calls

## Links
* no remix; pull this locally if you want to test.
* [Clubhouse](https://app.clubhouse.io/glitch/story/6236/ensure-consistent-casing-for-authtype-property-s-values-on-sign-in-link-clicked)

## GIF/Screenshots:
See [clubhouse comment](https://app.clubhouse.io/glitch/story/6236/ensure-consistent-casing-for-authtype-property-s-values-on-sign-in-link-clicked#activity-6271) for proof it worked

## Changes:
* Pass the authType as lowercase to match what dev tools does.

## How To Test:
* Run this branch locally; sign out of localhost if necessary. Use the network tab to block requests to `https://glitch.com*`, then sign in using a 3rd party auth provider. You'll be redirected to glitch.com/something-about-auth?token=XXXX; replace `glitch.com` with `localhost:3000` in the address bar and hit enter. Go to Segment's debugger and verify that a tracking event was created for your login, with auth type in lowercase.
